web_responsive/static/src/js/kanban_renderer_mobile.js

```diff
@@ -345,6 +345,7 @@
          *   and displayed
          */
         _moveToGroup: function (moveToIndex, animate) {
+            if (this.widgets.length == 0) return;
```

Comment:
Shouldn't you return with the promise resolved?

Reply:
Of course. My bad.